### PR TITLE
Fix patched version for nokogiri (GHSA-2qc6-mcvw-92cw)

### PR DESCRIPTION
**Pull request by GUI**
GHSA-2qc6-mcvw-92cw is reported to be fixed by 1.13.9+, not 1.13.19.

### CODE REVIEW

**Comment by alder-a** [comment]
Please merge this, as it is breaking our audit step in our build

**Review by alder-a — APPROVED** [review]

**Review by jondavidjohn — APPROVED** [review]

**Comment by aldent95** [comment]
Can this please be merged soon? I'm sure we are not the only ones having issues with their CI/CD yelling at them about failing bundle audits because of this issue.

**Review by aldent95 — APPROVED** [review]

**Comment by GSmes** [comment]
Merge, please! 🙏🏻

**Review by davidpaniz — APPROVED** [review]

**Review by markwilliamson503 — APPROVED** [review]

**Comment by carlwiedemann** [comment]
cc @reedloden @postmodern This is blocking CI for lots of people, it should be a trivial review/merge.

**Comment by JuanitoFatas** [comment]
Fyi: We could ignore this CVE to unblock CI until this gets merged

```
bundle-audit check --ignore GHSA-2qc6-mcvw-92cw
```

**Review by Rynaro — APPROVED** [review]

**Comment by aldent95** [comment]
> Fyi: We could ignore this CVE to unblock CI until this gets merged
> 
> ```
> bundle-audit check --ignore GHSA-2qc6-mcvw-92cw
> ```

We have tried that. Not sure if it's an issue with our build system but it's not working.

**Comment by rsanheim** [comment]
> Fyi: We could ignore this CVE to unblock CI until this gets merged

This worked for us on circleci FWIW:

```
bundle exec bundle-audit update && bundle exec bundle-audit check --ignore GHSA-2qc6-mcvw-92cw
```

**Review by reedloden — APPROVED** [review]

**Comment by royalpeasantry** [comment]
> > Fyi: We could ignore this CVE to unblock CI until this gets merged
> > ```
> > bundle-audit check --ignore GHSA-2qc6-mcvw-92cw
> > ```
> 
> We have tried that. Not sure if it's an issue with our build system but it's not working.

It also did not work for me until I updated the bundler-audit gem
```
bin/bundle update bundler-audit
...
Fetching bundler-audit 0.9.1 (was 0.6.1)
Installing bundler-audit 0.9.1 (was 0.6.1)
...
```

**Comment by reedloden** [comment]
Apologies to all for my typo! Sorry to any extra work this may have caused folks.

**Comment by flavorjones** [comment]
@reedloden Would it be helpful in the future if I shipped a PR here for the inevitable next nokogiri/libxml2 GHSA? Happy to do it, I've got a release checklist.

**Comment by radar** [comment]
@reedloden It's okay :) We all make mistakes. Thank you for your diligence on maintaining this repo.

**Comment by reedloden** [comment]
> @reedloden Would it be helpful in the future if I shipped a PR here for the inevitable next nokogiri/libxml2 GHSA? Happy to do it, I've got a release checklist.

Yes, please. That would be amazing!

**Comment by postmodern** [comment]
I bet we could add additional linter tests to check that every `>=` or `~>` version actually exists on rubygems.org.

**Pull request merged**
